doc/03_reference/cs_registers.rst

```diff
@@ -34,6 +34,8 @@ Ibex implements all the Control and Status Registers (CSRs) listed in the follow
 +---------+--------------------+--------+-----------------------------------------------+
 |  0x344  | ``mip``            | R      | Machine Interrupt Pending Register            |
 +---------+--------------------+--------+-----------------------------------------------+
+|  0x390  | ``mseccfg``        | RW     | Machine Security Configuration                |
```

Comment:
Is this specified as RW? it looks fairly WARLy to me

Reply:
As the epmp document isn't a proper spec I don't think it's defined as either! However WARL sounds like the sensible option, well spotted.